src/modules/alwaysontop/AlwaysOnTop/AlwaysOnTop.vcxproj

```diff
@@ -2,16 +2,6 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Project configurations -->
   <Import Project="..\..\..\..\packages\Microsoft.Windows.CppWinRT.2.0.200729.8\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\..\..\..\packages\Microsoft.Windows.CppWinRT.2.0.200729.8\build\native\Microsoft.Windows.CppWinRT.props')" />
-  <ItemGroup Label="ProjectConfigurations">
-    <ProjectConfiguration Include="Debug|x64">
-      <Configuration>Debug</Configuration>
-      <Platform>x64</Platform>
```

Comment:
just checking.. `<Platform>` not needed here

Reply:
I'm assuming not, that it will be applied by the build process, but I'll be giving this a spin in x64 to verify if things are working.

Reply:
The Platform config is handled in the `Cpp.Build.props` file.